### ISSUE TITLE
The profiler template "@OroCRM/Collector/crm.html.twig" does not exist.

### ISSUE THREAD

**Issue by vtsykun** [oroinc/crm#331]
I get error in dev mode (during profile load) for oro/crm 3.1.4 
```
The profiler template "@OroCRM/Collector/crm.html.twig" for data collector "oro_platform" does not exist.
```
![Selection_999(169)](https://user-images.githubusercontent.com/21358010/54380301-2ae4f000-469c-11e9-8d1e-63a2e82c0dbe.png)

It's related with commit https://github.com/oroinc/crm/commit/b574cc5024bd9c6c300e460b9dfa326bf6d8a566

**Comment by anyt** [comment]
Thanks for the report, internal ticket id #BAP-17423.
Will be fixed in next patch update.

**Comment by anyt** [comment]
Fixed at `4.0.0-beta`, and it will be included in ongoing `3.1.5`.
